cmd/argo/commands/get.go

```diff
@@ -18,6 +18,10 @@ import (
 
 const onExitSuffix = "onExit"
 
+var (
+	status string
```

Comment:
Instead of a global variable, we should make this a local variable to NewGetCommand() instead, and pass the status filter down to printNode() through function arguments.

Reply:
@jessesuen 

According to your comments,  I change the code and pass param through function arguments.

Pls take a look.  cc @sarabala1979